traits/trait_types.py

```diff
@@ -2577,9 +2575,8 @@ class Person(HasTraits):
 
     Parameters
     ----------
-    *values
-        Either all legal string values for the enumeration, or a single list
-        or tuple of legal string values.
+    values
+        A single list or tuple of legal string values.
```

Comment:
I'd be happy to modify this docstring not to advertise the `tuple` support; that support seems misplaced to me, and I'm not sure I want to promise such support going forward.